scss/style.scss

```
@@ -3,3 +3,261 @@
 @import "_expert.scss";
 @import "_works.scss";
 @import "_footer.scss";
+
+.header-wrapper {
+  height: 872px;
+  max-width: 100%;
+  background: url("../img/header_bg.jpg");
+
+  .img_header {
+    width: 147px;
+    height: 147px;
+    left: 926px;
+    top: 187px;
+    border-radius: 70px;
+    background: url("../img/guy.jpg");
+    margin: 150px auto 64px;
+    box-shadow: 0 0 5px 5px #b2ceff, 0 0 5px 15px #2c76f6;
```

Тенюшка нужна немного другая, не размытая
```suggestion
    box-shadow: 0 0 0 10px rgba($color: #fff, $alpha: 0.09),
      0 0 0 20px rgba($color: #fff, $alpha: 0.06);
```